pretext/pretext.py

```diff
@@ -3598,17 +3596,26 @@ def transform():
             # start will be reset to non-zero, so this is
             # one-time only, and never if there are no messages
             if (start == 0) and (end > 0):
-                outputfn("messages from the log for XSL processing (indented):")
+                log.info("messages from the log for XSL processing:")
             # print out any unprinted messages from error_log
-            for line in range(start, end):
-                outputfn(f"    * {xslt.error_log[line].message}")
+            for line in xslt.error_log[start:end]:
+                if "PTX:ERROR" in line.message:
+                    log.error(f"* {line.message}")
+                elif "PTX:WARNING" in line.message or "PTX:BUG" in line.message:
+                    log.warning(f"* {line.message}")
+                elif "PTX:FATAL" in line.message:
+                    log.critical(f"* {line.message}")
+                elif "PTX:DEBUG" in line.message:
+                    log.debug(f"* {line.message}")
+                else:
+                    log.info(f"* {line.message}")
```

Comment:
```suggestion
            for line in xslt.error_log[start:end]:
                if "PTX:FATAL" in line.message:
                    log.critical(f"* {line.message}")
                elif "PTX:ERROR" in line.message or "PTX:BUG" in line.message:
                    log.error(f"* {line.message}")
                elif "PTX:WARNING" in line.message:
                    log.warning(f"* {line.message}")
                elif "PTX:DEBUG" in line.message:
                    log.debug(f"* {line.message}")
                else:
                    log.info(f"* {line.message}")
```

Reply:
☝ as discussed in drop-in, `PTX:BUG` should be logged as an error. Also reordered in decreasing severity.